features/donations/quick_donation.feature

```diff
@@ -96,27 +96,45 @@ Scenario: admin logged in, records donation on behalf of patron
   Then I should see "Thank You for Your Purchase!"
   And customer "Joe Mallon" should have a donation of $9.00 to "General Fund"
   And an email should be sent to customer "Joe Mallon" containing "$  9.00  Donation to General Fund"
-  
+
 Scenario: landing on quick donation page with valid account code
+  Given I am logged in as customer "Tom Foolery"
   When I visit the quick donation landing page for account code 0504
   Then I should not see "Donate to"
   And I should see "Soda Fund"
   And I should see "The Soda Funds aims to put a Soda Fountain in Soda Hall Address"
 
 Scenario: landing on quick donation page with invalid account code
+  Given I am logged in as customer "Tom Foolery"
   When I visit the quick donation landing page for account code 0505
   Then I should see "Invalid Fund ID"
 
 Scenario: landing on quick donation page with no account code
+  Given I am logged in as customer "Tom Foolery"
   When I go to the quick donation page
   Then I should not see "Donate to"
   And I should see "General Fund"
```

Comment:
I point out that lines 116 and 117 are redundant.  if line 117 is true, line 116 will of necessity be true.  this is a big peril of using "Then I should see": it is true if the content occurs ANYWHERE on the page.  is there a way to scope this a bit more tightly, and if not, at least eliminate the redundant line?